example/app/views/layouts/_flash_messages.html.erb

```diff
@@ -4,11 +4,11 @@
 
     document.addEventListener(eventName, function() {
       <% if flash[:notice] %>
-        ShopifyApp.flashNotice("<%= escape_javascript flash[:notice].html_safe %>");
+        ShopifyApp.flashNotice("<%= flash[:notice].to_json %>");
```

Comment:
Remove the `""` surrounding `to_json`

Reply:
(see clayton's comments)

Reply:
And you'll need `<%==` at the start.

Reply:
Hey guys, sorry about the sloppiness. I've been working on something else, and this was kind of an after thought. Definitely thanks for taking a look!